main.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const ball = document.querySelector('.ball');
+const field = document.querySelector('.field');
+
+field.addEventListener('click', event => {
+  const ballWidth = ball.getBoundingClientRect().width;
+  const ballHeight = ball.getBoundingClientRect().height;
```

Comment:
getBoundingClientRect is hard operation for browser do it once for one element, destruct properties from result